### ISSUE TITLE
Create Text component

### ISSUE THREAD

**Issue by lorenzogm** [valtech-ch/ring#11]

**Comment by tamsler-valtech** [comment]
I don't think we need a title, when we have an RTE where the author can add titles.
The only thing one could think of is some variants to have some different CSS classes available to render the RTE differently, e.g. list numbering styles or maybe a background or even font style, though this could probably be done with the RTE directly.
